feat(docker): Add info about the docker deployment (#1098)
bonitasoft/bonita-doc: md/two-main-types-of-deployment.md

**Drawbacks**

* may not be adapted to some architecture constraints
## Bonita Portal + Bonita Engine running in a docker container

> This is a new deployment option available in the 7.10 version.

Reviewer: ```suggestion
This is a new deployment option available starting from 7.10 version.
```

Reviewer: Also the docker image was available in other version. It's just that we document it now

The installation process is described [here](bonita-docker-installation.md)

> **Advantages**

Reviewer 2: New line for formatting: new paragraph, not on the same line
```suggestion

**Advantages**
```

* The usual advantages provided by using a docker container : isolation, simplicity, scalability...
* Works out of the box with minimal effort.
* Is the natural solution for a cluster or cloud deployment

**Drawbacks**

* Probably overkill for a simple on premise single server installation

> * More complicated installation than the previous solution

Reviewer: I don't agree.
It's not overkill: it's more a matter of personal choice. If you already have some docker, it's easier to just run docker containers.
Also It's not more complicated installation (or it should not be). That's what you said in `Advantages`

IMHO, I would only put here something like `Require the installation of Docker`.

Reviewer 2: Agreed with @baptistemesta. I would not put **any** of those drawbacks, as they are not, for me:
* It's not overkill because docker is thought to be the same on an on-premise installation and "in the cloud".
* It's not more complicated to run

Reviewer: maybe we can put as drawback that it could be harder to customize the installation... but I'm not sure
